ScoutSuite/providers/gcp/resources/stackdriverlogging/sinks.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+from ScoutSuite.providers.base.configs.resources import Resources
+
+class Sinks(Resources):
+    def __init__(self, sdl_facade, project_id):
```

Comment:
I'm not a fan of the `sdl` abbreviation. It doesn't seem like it's commonly used at all. I think we should prefer long names, except for common abbreviations (GCP, EC2, ...)

Reply:
Gone!